Controller/Component/Cors/CorsComponent.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+class CorsComponent extends Component {
+
+	public function initialize(Controller $controller) {
+		parent::initialize($controller);
+		
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://dev.expandonline.nl:9000/sonarqube/coding_rules#rule_key=php%3AS1131)